docs/installation-places.md

```diff
@@ -1,35 +0,0 @@
-# Installation
```

Comment:
True there is a lot that is the same the main difference is the artifact name `mapzen-android-sdk` vs. `mapzen-places-api` but maybe some of the text only needs to be in there once.